setup.py

```diff
@@ -308,7 +308,8 @@ def _extract_libs_cffi_backend():
     "flake8",
     "hypothesis==4.5.5",
     "hypothesis-trio==0.3.0",
-    "black==18.9b0",  # Pin black to avoid flaky style check
+    "black==19.3b0",  # Pin black to avoid flaky style check
+    "flake8==3.7.7",
```

Comment:
`flake8` is already part of the list (L308)

Reply:
Fixed